README.md

```diff
@@ -67,7 +67,22 @@ first time starting the add-on).
 [![Docker Layers][layers-shield]][microbadger]
 [![Docker Pulls][pulls-shield]][dockerhub]
 
-## Configuration
+## Configuration Home assistant
+
+**Note**: _Remember to restart the add-on when the configuration is changed._
+
+**Note2**: _When changing from the official add-on to this one, make sure to change configuration.yaml broker ip. "broker: core-mosquitto" to "broker: localhost"_
+
+```json
+# Example configuration.yaml entry
+mqtt:
+  broker: localhost
```

Comment:
```suggestion
  broker: a0d7b954-mqtt
```

Reply:
as a newbie who is trying to connect this addon with HA. What's the meaning of `a0d7b954`  ?

Reply:
It's just part of the name to address the add-on: https://developers.home-assistant.io/docs/en/hassio_addon_communication.html#network